scripts/tables.sql

```diff
@@ -23,7 +23,9 @@ CREATE TABLE IF NOT EXISTS chat_history (
     user_message TEXT,
     assistant TEXT,
     message_time TIMESTAMP DEFAULT current_timestamp,
-    PRIMARY KEY (chat_id, message_id)
+    PRIMARY KEY (chat_id, message_id),
+    prompt_id UUID,
```

Comment:
prompt_id and brain_id should reference the prompts and brains table respectively, can be null